proto/umee/oracle/v1beta1/query.proto

```diff
@@ -10,24 +10,14 @@ option go_package = "github.com/umee-network/umee/x/oracle/types";
 
 // Query defines the gRPC querier service.
 service Query {
-  // ExchangeRate returns exchange rate of a denom
-  rpc ExchangeRate(QueryExchangeRateRequest) returns (QueryExchangeRateResponse) {
-    option (google.api.http).get = "/umee/oracle/v1beta1/denoms/{denom}/exchange_rate";
-  }
-
   // ExchangeRates returns exchange rates of all denoms
```

Comment:
nit: change the comment to explain that if specified it will return the exchange rate for a single denom instead of all.

Reply:
https://github.com/umee-network/umee/pull/264/commits/2a7c1fdacfaf9da98cb239cc82e7561cf3b86f93 👍